frontend/src/pages/Visitors.vue

```diff
@@ -118,6 +150,10 @@
                 };
                 this.data.push(item);
             }
+            this.pieData.push(
```

Comment:
not quite sure why you need updating pieData if you can define this in `data`

Reply:
Fixed